modules/seqexec/server/src/main/scala/seqexec/server/gnirs/GnirsControllerEpics.scala

```diff
@@ -295,11 +300,19 @@ object GnirsControllerEpics extends GnirsController {
       GnirsEpics.instance.abortCmd.mark *>
       GnirsEpics.instance.abortCmd.post.void
 
-  override def observeProgress(total: Time): Stream[IO, Progress] =
+  private def removePartName(s: String) = {
```

Comment:
I think this was changed upstream. It is now at
https://github.com/gemini-hlsw/ocs3/blob/develop/modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala#L271

Reply:
Ok thanks I'll have a look.